pkg/features/kube_features.go

```diff
@@ -752,6 +752,13 @@ const (
 	// Enables topology aware hints for EndpointSlices
 	TopologyAwareHints featuregate.Feature = "TopologyAwareHints"
 
+	// owner: @kannon92
+	// kep: https://kep.k8s.io/3939
+	// alpha: v1.28
+
+	// Allow customizing pod replacement policy for Jobs.
+	JobPodReplacementPolicy featuregate.Feature = "JobPodReplacementPolicy"
```

Comment:
Move it to the other Job features alphabetically.

Reply:
done.